drivers/include/saul.h

```diff
@@ -105,6 +105,7 @@ enum {
     SAUL_SENSE_CAPACITANCE = 0x97,     /**< sensor: capacitance */
     SAUL_SENSE_VOLTAGE     = 0x98,     /**< sensor: voltage */
     SAUL_SENSE_PH          = 0x99,     /**< sensor: pH */
+    SAUL_SENSE_POWER       = 0x9a,     /**< sensor: (electrical) power */
```

Comment:
Is it necessary to specify that electrical power is meant? If not, we could drop it. If yes, I wouldn't place it in braces.

Reply:
You're totally right. The physical unit will be Watt regardless of what power we will speak and having the electrical written there will trigger new entries for each other kind of power, even though there is no reason to differentiate.

Reply:
That sensor type was added to master in the meantime with the same comment. I changed the commit to add this during rebasing to a commit that removes the `(electrical)`.